java/src/jmri/jmrix/powerline/DmxSequence.java

```diff
@@ -0,0 +1,85 @@
+package jmri.jmrix.powerline;
+
+/**
+ * Represent a sequence of one or more Dmx commands (unit and intensity).
+ * <p>
+ * These are Dmx specific, but not device/interface specific.
+ * <p>
+ * A sequence should consist of addressing (1 or more), and then one or more
+ * commands. It can address multiple devices.
+ *
+ * @author Bob Coleman Copyright (C) 2010
+ * @author Bob Jacobsen Copyright (C) 2008
+ * @author Ken Cameron Copyright (C) 2023
+ */
+public class DmxSequence {
+
+    // First implementation of this class uses a fixed length
+    // array to hold the sequence; there's a practical limit to how
+    // many Dmx commands anybody would want to send at once!
+    private static final int MAXINDEX = 32;
+    int index = 0;
+    Command[] cmds = new Cmd[MAXINDEX];  // doesn't scale, but that's for another day
```

Comment:
It's preferred to have fields private if possible. That ensures that it's only this class that can change them. It also makes it much easier if you later want to change them, for example changing the array to a list.